cumulus/polkadot-parachain/polkadot-parachain-lib/src/command.rs

```diff
@@ -297,17 +302,20 @@ pub fn run<CliConfig: crate::cli::CliConfig>(cmd_config: RunConfig) -> Result<()
 	}
 }
 
+/// This method is needed only for `#[sc_tracing::logging::prefix_logs_with("Parachain")]`.
+///
+/// Adding this annotation to `DynNodeSpec::start_node()` doesn't work, probably because of the
```

Comment:
If we want to get rid of this, you could try to add
```rust
.instrument(sc_tracing::tracing::info_span!(
sc_tracing::logging::PREFIX_LOG_SPAN,
name = "Parachain",
)))
```
to the futures that are being run. This is what this macro should also be doing.